application/include/mmul.hpp

```diff
@@ -18,15 +18,16 @@ using namespace Eigen;
 
 template <typename Executor,
           typename execution::instance_of_base<inline_executor, Executor> = 0>
-void mmul(Executor ex, MatrixXd &a, MatrixXd &b, MatrixXd &c) {
+void mmul(const Executor ex, const MatrixXd &a, const MatrixXd &b,
+          MatrixXd &c) {
   auto mul = [&]() { c = a * b; };
 
   ex.execute(mul);
 }
 
 template <typename Executor,
           typename execution::instance_of_base<omp_executor, Executor> = 0>
-void mmul(Executor ex, MatrixXd &a, MatrixXd &b, MatrixXd &c) {
+void mmul(const Executor ex, MatrixXd &a, MatrixXd &b, MatrixXd &c) {
```

Comment:
`a` and `b` missing `const`? or is something preventing it?

Reply:
I overlooked it :sweat: